packages/color-block/src/helpers/mapColorSpaces.ts

```diff
@@ -0,0 +1,189 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { FrontifyColor } from '@frontify/app-bridge';
+
+type ColorSpace = {
+    id: string;
+    key?: string;
+    label?: string;
+    value?: string;
+    placeholder?: string;
+};
+
+export const mapColorSpaces = (colorSpaceID: string, color?: FrontifyColor) => {
+    let mappedColorSpace: ColorSpace = { id: colorSpaceID };
+
+    switch (colorSpaceID) {
+        case 'hex':
+            mappedColorSpace = {
+                id: colorSpaceID,
+                label: 'HEX',
+                value: color && `#${color.hex}`,
+                placeholder: '#hexhex',
+            };
```

Comment:
```suggestion
            return {
                id: colorSpaceId,
                label: 'HEX',
                value: `#${color?.hex}`,
                placeholder: '#hexhex',
            };
```
You can do the same everywhere else:
- Remove the `mappedColorSpace =`
- `colorSpaceID` to `colorSpaceId`
- Remove the `color &&` and prefer `color?.` (except when using multiple times `color` => rgb, cmyk, ...)